packages/stryker/bin/stryker

```diff
@@ -5,4 +5,6 @@
 process.title = 'stryker';
 
 // Run the TypeScript compiled stryker-cli.js
-require('../src/stryker-cli');
\ No newline at end of file
+//require('../src/stryker-cli');
+const StrykerCli = require('../src/StrykerCli');
```

Comment:
The constructor function that you export in the StrykerCli file is added to the property `default` so I believe this line should be `const StrykerCli = require('../src/StrykerCli').default;`